tox.ini

```diff
@@ -0,0 +1,13 @@
+# Tox (http://tox.testrun.org/) is a tool for running tests
+# in multiple virtualenvs. This configuration file will run the
+# test suite on all supported python versions. To use it, "pip install tox"
+# and then run "tox" from this directory.
+
+[tox]
+envlist = py27
+
+[testenv]
+commands = make test
```

Comment:
@bitglue the only changes I made to your PR https://github.com/paperg/tagcompare/pull/4 is in this file here.  Had to add the PYTHONPATH env to make it work and changed the commands to `make test` for now until https://github.com/paperg/tagcompare/issues/8